clients/pkg/promtail/targets/gcplog/formatter.go

```diff
@@ -53,11 +53,12 @@ func format(
 
 	// mandatory label for gcplog
 	lbs := labels.NewBuilder(nil)
-	lbs.Set("resource_type", ge.Resource.Type)
+	lbs.Set("__gcp_logname", ge.LogName)
+	lbs.Set("__gcp_resource_type", ge.Resource.Type)
```

Comment:
@james-callahan This is a breaking change!. 

Can you please record it in the `upgrading/_index.md`?

Reply:
I thought that was usually done after merge?

Reply:
You can add it in the same PR :)